app/src/main/java/org/simple/clinic/contactpatient/views/CallPatientView.kt

```diff
@@ -147,7 +150,17 @@ class CallPatientView(
     patientAddressTextView.text = patientDetails.patientAddress
     registeredFacilityTextView.text = patientDetails.registeredFacility
     diagnosisTextView.text = diagnosis
-    lastVisitedTextView.text = dateTimeFormatter.format(patientDetails.lastVisited?.toLocalDateAtZone(userClock.zone))
+    isLastSeenAvailable(dateTimeFormatter, patientDetails, userClock)
+  }
+
+  private fun isLastSeenAvailable(dateTimeFormatter: DateTimeFormatter, patientDetails: PatientDetails, userClock: UserClock) {
```

Comment:
The prefix `is` for the method implies that the method returns a `Boolean`, but here we are changing the state of the UI based on some conditions. Can we update the method name to better reflect this, please?